testng-core/src/test/java/test/reports/EmailableReporterTest.java

```diff
@@ -4,32 +4,16 @@
 
 import java.io.File;
 import java.lang.reflect.Method;
-import java.security.Permission;
 import org.testng.IReporter;
 import org.testng.ITestNGListener;
 import org.testng.TestNG;
-import org.testng.annotations.AfterClass;
-import org.testng.annotations.BeforeClass;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 import org.testng.reporters.EmailableReporter;
 import org.testng.reporters.EmailableReporter2;
 import test.SimpleBaseTest;
 
 public class EmailableReporterTest extends SimpleBaseTest {
```

Comment:
Same goes for this as well. Any specific reason behind us removing off this test that works with a custom security manager?

Reply:
I removed it because of https://openjdk.java.net/jeps/411 .

Reply:
@hduerkop - Please revert it back since we are still saying that we are going to be JDK11 as the minimum JDK but the security manager has been deprecated in JDK17. So we should perhaps leave it there as is until TestNG moves into JDK17 or above as its minimum JDK.

Reply:
I have to disagree here since JDK17 is the current LTS and is or soon will be dominant on build platforms

Reply:
@hduerkop - if testng has jdk11 as its minimum jdk then we will need to have everything that exists in jdk11 including tests that test features which are relevant for jdk11 as well. 

I agree that build systems atleast on the open source world may move forward but we need to be cognizant of user needs and cannot enforce this upon our users.

Reply:
It is not an enforcement, but an option to use JDK17 this way - 11 and 17 work this way.

Reply:
What was the initial objective of the test? Is it another way to test it?